### DISCUSSION TITLE
Braille Patterns

### DISCUSSION THREAD

**Discussion by dfballes** [psb1558/Junicode-font#168] [Ideas]
Hi,

would it be possible to add the Braille Patterns (U+2800 to U+28FF) in Junicode? It is my favorite font and I would really like to work with it for a student of mine, which is visually impaired but tries to learn Music-Braille.

Greetings,
Daniel

**Comment by psb1558** [comment]
I haven't been ignoring this post, but thinking about it. My view, in the end, is that I could add very little of value by covering the Unicode Braille range, since a Junicode Braille would be like everybody else's, and Braille is well covered by excellent free fonts like, for example, the [Deja Vu family](https://dejavu-fonts.github.io/). In whatever software you use, it is dead easy to set up a style that will switch into Deja Vu (or whatever you like) and very likely perform other necessary transformations (such as mapping the abstract patterns to particular scripts).

**Comment by dfballes** [comment]
Thank you nonetheless, Peter. I already thought that it would be that way, but wanted to propose the idea anyways. Now I just have to figure out how Word is handling the font substitutions, since my student does not have every font installed on his PC. But that is my problem ;-)
